recipe/meta.yaml

```
@@ -53,22 +54,20 @@ outputs:
         - {{ pin_subpackage("libabseil", max_pin="x") }}
         # also pin on ABI variant
         - libabseil =*=cxx{{ cxx_standard }}*
-        # abseil needs the 10.13 headers even just to be `# include`d;
-        # ensure that all dependent packages reflect that to
-        # avoid the solver getting stuck in no-man's-land.
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
 
```

we don't want the run-export from gtest for libabseil itself
```suggestion
      ignore_run_exports:
        - gtest

```

---

Updated as requested